samples/cpp/benchmark_app/property_map.hpp

```diff
@@ -0,0 +1,172 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/runtime/intel_gna/properties.hpp>
+#include <openvino/runtime/intel_gpu/properties.hpp>
+#include <openvino/runtime/intel_gpu/remote_properties.hpp>
+#include <openvino/runtime/intel_myriad/hddl_properties.hpp>
+#include <openvino/runtime/intel_myriad/myriad_properties.hpp>
+
+#define VALUE_2_STRING(name) #name
+#define INSERT_NAME_VALUE_MAP(out_map, value)        \
+    {                                                \
+        std::string str_key = VALUE_2_STRING(value); \
+        std::stringstream str_value;                 \
+        str_value << value;                          \
+        out_map[str_key] = str_value.str();          \
+    }
+
+enum PropertyType {
+    PROPERTY_TYPE_STRING = 0,
+    PROPERTY_TYPE_INTER,
+    PROPERTY_TYPE_UNSIGNED_INTER,
+    PROPERTY_TYPE_INTER64,
+    PROPERTY_TYPE_ENUM,
+    PROPERTY_TYPE_BOOL,
+    PROPERTY_TYPE_FLOAT,
+    PROPERTY_TYPE_STRING_FLOAT_MAP,
+};
+
+struct PropertyInfo {
+    std::string property_name;
+    PropertyType property_type;
+};
+
+#define INSERT_PROPERTY_MAP(out_map, value, type, reversed) \
+    {                                                       \
+        std::string str_key = VALUE_2_STRING(value);        \
+        std::string str_name = value.name();                \
+        PropertyInfo property_info;                         \
+        property_info.property_type = type;                 \
+        if (false == reversed) {                            \
```

Comment:
```suggestion
        if (!reversed) {                            \
```
However there's `else` section so better to place it other way round - positive comes first.

Reply:
Yes, I will fix it.